docs/VictoriaLogs/README.md

```diff
@@ -23,263 +23,18 @@ VictoriaLogs is at Preview stage now. It is recommended evaluating it on product
 It isn't recommended migrating from existing logging solutions to VictoriaLogs Preview in general case yet.
 See the [Roadmap](https://docs.victoriametrics.com/VictoriaLogs/Roadmap.html) for details.
 
-## Operation
-
 ### How to run VictoriaLogs
 
-There are the following options exist now:
-
-- To run Docker image:
-
-  ```bash
-  docker run --rm -it -p 9428:9428 -v ./victoria-logs-data:/victoria-logs-data \
-    docker.io/victoriametrics/victoria-logs:heads-public-single-node-0-ga638f5e2b
-  ```
-
-- To build VictoriaLogs from source code:
-
-  Checkout VictoriaLogs source code. It is located in the VictoriaMetrics repository:
-
-  ```bash
-  git clone https://github.com/VictoriaMetrics/VictoriaMetrics
-  cd VictoriaMetrics
-  ```
-
-  Then build VictoriaLogs. The build command requires [Go 1.20](https://golang.org/doc/install).
-
-  ```bash
-  make victoria-logs
-  ```
-
-  Then run the built binary:
-
-  ```bash
-  bin/victoria-logs
-  ```
-
-VictoriaLogs is ready to [receive logs](#data-ingestion) and [query logs](#querying) at the TCP port `9428` now!
-It has no any external dependencies, so it may run in various environments without additional setup and configuration.
-VictoriaLogs automatically adapts to the available CPU and RAM resources. It also automatically setups and creates
-the needed indexes during [data ingestion](#data-ingestion).
-
-It is possible to change the TCP port via `-httpListenAddr` command-line flag. For example, the following command
-starts VictoriaLogs, which accepts incoming requests at port `9200` (aka ElasticSearch HTTP API port):
-
-```bash
-/path/to/victoria-logs -httpListenAddr=:9200
-```
-
-VictoriaLogs stores the ingested data to the `victoria-logs-data` directory by default. The directory can be changed
-via `-storageDataPath` command-line flag. See [these docs](#storage) for details.
-
-By default VictoriaLogs stores log entries with timestamps in the time range `[now-7d, now]`, while dropping logs outside the given time range.
-E.g. it uses the retention of 7 days. Read [these docs](#retention) on how to control the retention for the [ingested](#data-ingestion) logs.
-
-It is recommended setting up monitoring of VictoriaLogs according to [these docs](#monitoring).
+See [Quick Start](https://docs.victoriametrics.com/VictoriaLogs/quickstart.html) for getting started with VictoriaLogs.
```

Comment:
Could you please define the separate section and Call it Quickstart - we definitely will want to provide the anchored link to it